symbolic-base/src/ZkFold/Base/Protocol/Plonkup/Prover.hs

```diff
@@ -159,22 +161,25 @@ plonkupProve PlonkupProverSetup {..}
         gammaX   = polyVecConstant gamma
         deltaX   = polyVecConstant delta
         epsilonX = polyVecConstant epsilon
+
+        -- TODO: even with all these @force@-s, @qX@ still generates somewhat big thunks, most of which are from @genericDft@.
+        -- If this becomes critical in the future, we'll need to add strict multiplication.
```

Comment:
I think it's just that multiplication of polynomials (`PolyVec`-s, I guess?) has to be strict

Reply:
Yeah, sure.